swan-lake/development-tutorials/ballerina-central/configure-proxy.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: ballerina-publishing-to-central-left-nav-pages-swanlake
+title: Configure proxy
+description: Below section describes how to configure proxy for accessing Ballerina Central, or custom repositories
+keywords: ballerina, programming language, configuring proxy, proxy, set-up proxy
+permalink: /learn/configure-proxy/
+active: configure-proxy
+intro: Below section describes how to configure proxy for accessing Ballerina Central, or custom repositories
+---
+
+## Configure proxy
+
+If you are connected to the internet via an HTTP proxy, you need to configure it in the `Settings.toml` file to carry out the Ballerina Central related operations such as publishing a package, pulling a package or resolving packages. Add the following section to `Settings.toml` and change accordingly.
+
+```toml
+[proxy]
+host = "localhost"
+port = 3128
+username = "solomon"
+password = "confidential"
+```
```

Comment:
IMO, it is better to have general place holders rather than example values in the documentation. wdyt?
```suggestion
If you are connected to the internet via an HTTP proxy, you need to configure it in the `Settings.toml` file to carry out the Ballerina Central related operations such as publishing a package, pulling a package or resolving packages. Add the following section to `Settings.toml`.

```toml
[proxy]
host = "<HOST_NAME"
port = <PORT>
username = "<PROXY_USERNAME>"
password = "<PROXY_PASSWORD>"
```
```